js/src/personal/riyuan/components/about/About.tsx

```diff
@@ -37,7 +37,16 @@ function InfoSection({ title, description, listItems }: InfoSectionProps) {
         size="sm"
       >
         {listItems.map((item, index) => (
-          <ListItem key={index}>{item}</ListItem>
+          <ListItem key={index}>
+            <Text
+              ta="left"
+              size="md"
+              mx="auto"
+              pl={5}
+            >
+              {item}
```

Comment:
nit: "item" is so generic I have to read the prop type above to know what this is.

Reply:
change it to bulletPoint